ci/python_bandit_check.sh

```diff
@@ -3,7 +3,14 @@
 set -uex
 
 git clean -dxf
+# Nothing seems to work to filter the submodules but as they are separate
+# projects, things like bandit should be in those projects, not in DAOS.
```

Comment:
`bandit --exclude` doesn't work?

Reply:
I tried everything and never got it to work